client/src/components/InfoPanel.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {
+  BoldBox, Container, GrayWrapper, InfoContainer, ModuleWrapper, LinkWrapper
+} from './styles/styles';
+import moment from 'moment';
```

Comment:
I'm not super familiar with moment and what it does but looks like you have the hang of it and are implementing it well

Reply:
Update: it looks like its used to format dates - pretty cool!